pkgs/development/python-modules/triton/binary-hashes.nix

```diff
@@ -7,26 +7,31 @@
 
 version:
 builtins.getAttr version {
-  "2.1.0" = {
+  "3.0.0" = {
     x86_64-linux-38 = {
-      name = "triton-2.1.0-cp38-cp38-linux_x86_64.whl";
-      url = "https://download.pytorch.org/whl/triton-2.1.0-0-cp38-cp38-manylinux2014_x86_64.manylinux_2_17_x86_64.whl";
-      hash = "sha256-Ofb7a9zLPpjzFS4/vqck8a6ufXSUErux+pxEHUdOuiY=";
+      name = "triton-3.0.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl";
+      url = "https://files.pythonhosted.org/packages/ae/b2/048c9ecfdba0e6b0ae3c02eed2d9dd3e9e990a6d46da98555cf0c2232168/triton-3.0.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl";
+      hash = "sha256-6JA3Z5Ub+G7JYLT+TiG8lwBVr8ZenVfpFtea48k2ZeM=";
     };
     x86_64-linux-39 = {
-      name = "triton-2.1.0-cp39-cp39-linux_x86_64.whl";
-      url = "https://download.pytorch.org/whl/triton-2.1.0-0-cp39-cp39-manylinux2014_x86_64.manylinux_2_17_x86_64.whl";
-      hash = "sha256-IVROUiwCAFpibIrWPTm9/y8x1BBpWSkZ7ygelk7SZEY=";
+      name = "triton-3.0.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl ";
+      url = "https://files.pythonhosted.org/packages/ea/6b/1d72cc8a7379822dadf050474add7d8b73b02c35057446b6f17d27cb9ea2/triton-3.0.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl";
+      hash = "sha256-QQBPsa6aU/yz6XB0X+uH8OPJTGzhuobpX6O4U3iUvvc=";
     };
     x86_64-linux-310 = {
-      name = "triton-2.1.0-cp310-cp310-linux_x86_64.whl";
-      url = "https://download.pytorch.org/whl/triton-2.1.0-0-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl";
-      hash = "sha256-ZkOZI6MNXUg5mwip6uEDcPbCYaXshkpkmDuuYxUtOdc=";
+      name = "triton-3.0.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl ";
+      url = "https://files.pythonhosted.org/packages/15/67/84e5a4b7b45bdeb11da26a67dfa2b988c512abbcbcad8cbc30aa579051b2/triton-3.0.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl";
```

Comment:
Right, about this 1, as I mentioned in https://github.com/NixOS/nixpkgs/pull/328247 I am concerned about changing provenance. We used to download from one party (pytorch) now we're downloading from another (pypi) which was just recently specifically exploited in the context of pytorch.

Did pytorch stop distributing triton wheels or what's the situation here?

Reply:
i think so

Reply:
nvm, I just checked, they added the new version now too. I'll edit my commit.

Reply:
I'm done with the changes, gonna commit tomorrow.